src/main/java/blockchain/LightChainNode.java

```diff
@@ -700,6 +706,34 @@ public boolean isSound(Transaction t) {
 		}
 	}
 
+	
+		public boolean isCorrect(Transaction t, Contract c) throws IOException { 
+			try {
+
+				TransferTest tesq;
+				tesq = new TransferTest();
```

Comment:
You can shorten these two lines into:
`TransferTest tesq = new TransaferTest();`